examples/generic-azure.yaml

```diff
@@ -673,3 +673,26 @@ groups:
           ## References
           - section 9.1.2 https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
         scored: true
+      - id: 1.5.3
+        description: "<sid>adm user and its <sid> group exist and enabled"
+        audit: |
+          sid=$(sudo crm configure show | grep -m1 SID= | sed -e "s/.*SID=\(...\).*/\1/" | tr '[:upper:]' '[:lower:]')
+          if cat /etc/passwd | cut -d: -f1 | grep ${sid}adm; then echo userexists; fi
+          if ! cat /etc/passwd | grep -e "^${sid}adm" | cut -d: -f7 | grep "nologin"; then echo userenabled; fi
```

Comment:
We also need to check other ways to "disable" users. (eg first character disables the user).

Reply:
> We also need to check other ways to "disable" users. (eg first character disables the user).

Could you please give more details, what do you mean by disabling a user by a first character?